perfmetrics/scripts/continuous_test/ml_tests/tf/readme.txt

```diff
@@ -0,0 +1,15 @@
+This file contains the code flow for tf based kokoro job.
+
+The Kokoro vm will start with build.sh, setting up the vm with nvidia drivers 
```

Comment:
Please restructure the readme file to have the following sections:
1. Test description. Also, specify model used, training data, epochs, duration etc.
2. Packages/libraries required for test including their short description. 
3. Description and function of each file used in this test.
4. About logs produced in this test including location.
5. Where to make changes in code for changing things in test like epochs, batch size, tf version etc.